kola-denylist.yaml

```
@@ -47,3 +47,9 @@
   osversion:
     - centos-10
     - rhel-10.1
+
+- pattern:
+  tracker: https://github.com/coreos/rhel-coreos-config/issues/93
+  osversion:
+    - rhel-10.1
+    - rhel-9.6
```

```suggestion
    - rhel-10.1
```

We can ignore tests on 10.1 for some time, but we can't really on 9.6. I realize this means this PR can't merge until we get a newer ignition RPM, but I think that's a hard requirement at this point.